src/app/modules/vault/vault.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from "@angular/core";
+
+import { LooseComponentsModule } from "../loose-components.module";
+import { SharedModule } from "../shared.module";
+import { VaultFilterModule } from "../vault-filter/vault-filter.module";
+
+import { VaultService } from "./vault.service";
+
+@NgModule({
+  imports: [SharedModule, VaultFilterModule, LooseComponentsModule],
+  exports: [SharedModule, VaultFilterModule, LooseComponentsModule],
+  providers: [
+    {
+      provide: VaultService,
+      useClass: VaultService,
+    },
+  ],
+})
+export class VaultModule {}
```

Comment:
Feels like this should be a module under `organizations`, and not vault.

Reply:
Assuming you mean you'd want `OrganizationVaultModule` to be a `VaultModule` underneath an `OrganizationModule` or something like that:

I'm open to discussing this, because I understand that the current components are organized this way and I am sure there is a good reason for it, but in my opinion this is a flaw in the application structure. These Vault components have *WAY* more in common with each other than they do with the larger concepts of organizations, or lack there of. They should be near each other in the file system to more easily distinguish that similarity and to have access to shared code without deep relative imports to far away areas of the file structure. 

The `OrganizationsModule` (theoretically, since this doesn't exists) would still only import the `OrganizationVaultModule`, the `VaultModule` itself is only intended to be consumed by its children and does not declare or export any of its children.